api/api.proto

```diff
@@ -2512,6 +2512,15 @@ service OpenStorageVolume {
           body: "*"
         };
     }
+
+    // Watch streams a list of volumes exist in the system.
+    rpc Watch(SdkVolumeWatchRequest)
```

Comment:
This is good, but I would suggest creating a model of Watch for anything on the system, not in the Volumes service. That way we can watch for Pool, Disk, Volume, or any other type of event. Then the event message could use `oneof` for each type of storage event.